research/H1/CFB.md

```
@@ -159,5 +167,5 @@ Full Preamble | Description
 ------------- | -----------
 [#CFB.01](CFB/CFB.01.md) | Failure/fault/warning
 #CFBFLR | Realtime Failure
-#CFBMPF |
+#CFBMPF | Maintenance Planning Function
```

markdownlint-cli2 (0.21.0)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)